### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pjanik**
README.md updates that follow a set of requirements listed here:
https://www.pivotaltracker.com/story/show/181051946

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +37,50 @@ You *do not* need to build to deploy the code, that is automatic.  See more info
...
-## Deployment
+## Deployment to Concord Consortium AWS S3 bucket (https://soundwaves.concord.org)
 
 Production releases to S3 are based on the contents of the /dist folder and are built automatically by GitHub Actions
 for each branch pushed to GitHub and each merge into production.
 
-Merges into production are deployed to http://soundwaves.concord.org.
+Merges into production are deployed to https://soundwaves.concord.org.
+
+Other branches are deployed to `https://soundwaves.concord.org/branch/<name>/index.html`, for example: https://soundwaves.concord.org/branch/master/index.html.
```
Comment (kswenson) [inline README.md:47]:
```suggestion
Other branches are deployed to `https://soundwaves.concord.org/branch/<name>/`, for example: https://soundwaves.concord.org/branch/master/ (note the trailing slash).
```

File: README.md [hunk 2]
```diff
@@ -34,14 +37,50 @@ You *do not* need to build to deploy the code, that is automatic.  See more info
...
+recording, downsampling for graphing needs, and even AM and FM modulation.
+
+MDN provides a great documentation and tutorials:
+https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API
+
+Most of the custom audio-related helper are implemented in `src/utils/audio.ts`.
+
+AM and FM modulation is implemented using `OfflineAudioContext`, `OscillatorNode`, and `GainNode`.
+
+## Know issues
```
Comment (kswenson) [inline README.md:78]:
```suggestion
## Known issues
```

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Looks good -- I made a few minor suggestions, one being to include the Webkit tracking info here.

File: README.md (hunk 1 above)
Comment (pjanik) [reply README.md:47]: I didn't know someone made that work, nice. 👍 
Interesting, when I try `https://soundwaves.concord.org/branch/master`, it redirects to `https://soundwaves.concord.org/soundwaves/branch/master/` (which is ofc 404).

**Comment by pjanik** [comment]
Thanks. I've merged all your suggestions, so merging the whole PR now.

File: README.md (hunk 1 above)
Comment (kswenson) [reply README.md]: > I didn't know someone made that work, nice. 👍 Interesting, when I try `https://soundwaves.concord.org/branch/master`, it redirects to `https://soundwaves.concord.org/soundwaves/branch/master/` (which is ofc 404).

That trailing slash requirement has been an annoyance for quite some time now but no one's ever gotten around to doing anything about it. 😉

**Pull request merged**